api/src/prescription/organization-place/application/organization-place-route.js

```diff
@@ -149,7 +149,7 @@ const register = async (server) => {
       method: 'GET',
       path: '/api/data/organization-places',
       config: {
-        auth: 'jwt-pix-data',
+        auth: { strategy: 'jwt-application', access: { scope: ['statistics'] } },
```

Comment:
suggestion(non-bloquante): Juste pour être cohérent avec les autres déclarations
```suggestion
        auth: { strategy: 'jwt-application', access: { scope: 'statistics' } },
```